.travis.yml

```diff
@@ -1,8 +1,8 @@
-sudo: required
-language: bash
-
-services:
-  - docker
+language: go
 
 script:
   - make all
+  
+after_success:
+  - bash generate-coverage.sh
```

Comment:
But let's run it outside of the travis stuff since the code coverage will be go, or ruby, specific depending on which sub-project we're building/testing.  Moving it into the k8s portion will allow us to run the go version for just the k8s side of things.